sound/soc/intel/common/soc-acpi-intel-adl-match.c

```diff
@@ -490,6 +495,13 @@ struct snd_soc_acpi_mach snd_soc_acpi_intel_adl_machines[] = {
 		.drv_name = "adl_cs35l41",
 		.sof_tplg_filename = "sof-adl-cs35l41.tplg",
 	},
+	{
+		.comp_ids = &essx_83x6,
+		.drv_name = "sof_es83x6_hdmi_ssp",
```

Comment:
use a better name that is more representative of your board. this says nothing about hdmi capture and ssp0.

Reply:
We have modified the name sof_es83x6_hdmi_ssp -> adl_es83x6_hdmi_ssp. Due to the character limit for drv_name, we could not include more information like usage of ssp0 and ssp2.